nbs/explains/minidataapi.ipynb

```diff
@@ -0,0 +1,516 @@
+{
```

Comment:
_Commented on notebook `nbs/explains/minidataapi.ipynb` Cell 19 Line 1_
```python
for user in users():
```
Same line here too -- or better still just say `users()` and that's it. No need for `print` or a loop afaict